lib/fluent/plugin/s3_compressor_arrow.rb

```diff
@@ -45,11 +45,13 @@ def compress(chunk, tmp)
         stream = Arrow::BufferInputStream.new(buffer)
         table = Arrow::JSONReader.new(stream, @options)
 
-        table.read.save(tmp,
+        save_options = {
           format: @arrow.format,
-          chunk_size: @arrow.chunk_size,
           compression: @arrow.compression,
-        )
+        }
+        save_options[:chunk_size] = @arrow.chunk_size if @arrow.chunk_size
+        
+        table.read.save(tmp,save_options)
```

Comment:
Could you confirm whether this hunk is needed or not?
I think that `chunk_size: @arrow.chunk_size` where `@arrow.chunk_size` is `nil` should work.

Reply:
After checking the code, it works well if we pass chunk_size with a value of nil. So this PR change is completed only by changing the default size in config_pram. Thanks!